install/0000_30_machine-api-operator_01_images.configmap.yaml

```diff
@@ -11,5 +11,7 @@ data:
       "clusterAPIControllerOpenStack": "docker.io/openshift/origin-openstack-machine-controllers:v4.0.0",
       "clusterAPIControllerLibvirt": "docker.io/openshift/origin-libvirt-machine-controllers:v4.0.0",
       "clusterAPIControllerBareMetal": "quay.io/openshift/origin-baremetal-machine-controllers:v4.0.0",
-      "clusterAPIControllerAzure": "quay.io/openshift/origin-azure-machine-controllers:v4.0.0"
+      "clusterAPIControllerAzure": "quay.io/openshift/origin-azure-machine-controllers:v4.0.0",
+      "baremetalOperator": "quay.io/openshift/origin-baremetal-operator:latest",
+      "baremetalProvider": "quay.io/repository/metal3-io/ironic:latest"
```

Comment:
`baremetalProvider` doesn't appear to be used anywhere

Reply:
baremetalProvider holds the Ironic image. It would be used when we move to using  https://github.com/metal3-io/baremetal-operator/blob/master/deploy/operator_ironic.yaml to lauch the baremetal-operator pod. Will post an update to these patches as soon as the discussion involving the best operator to deploy the baremetal-operator pod is resolved.